src/components/Envelope.vue

```diff
@@ -1072,13 +1072,19 @@ export default {
 }
 :deep(.list-item) {
 	flex-wrap: wrap;
+	padding: var(--default-grid-baseline);
```

Comment:
Could you move this into the skeleton component? list item is our own component now. we don't need the deep selectors to patch styles